extensions/interactions/FractionInput/directives/fraction_input_interaction_directive.html

```diff
@@ -1,7 +1,7 @@
-<form ng-disabled="FractionInputForm.$invalid" ng-submit="submitAnswer(answer)" role="form" class="form-horizontal">
+<form ng-disabled="FractionInputForm.$invalid" ng-submit="submitAnswer(answer)" role="form" class="form-horizontal fraction-input-interaction">
   <!-- This ng-form is needed for submitting answer on hitting enter. -->
   <ng-form name="FractionInputForm">
-    <input type="text" name="answer" ng-model="answer" class="form-control"
+    <input type="text" name="answer" ng-model="answer" class="form-control fraction-input-interaction"
```

Comment:
Er, I didn't mean add a fraction-input-interaction class, and it also doesn't make sense to use the same class to label two different things nested within each other.

I meant, can you check what directive "wraps" this template in the DOM by using "Inspect Element" in the browser? If there's a custom tag name containing this then you can prefix your CSS selectors by that custom tag name (no dot). See e.g. components/response_header_directive.html, note how each CSS selector is prefixed by response-header to scope it to that custom tag only.